src/openssl.c

```diff
@@ -4512,6 +4553,7 @@ static const auxL_Reg pk_methods[] = {
 	{ "verify",        &pk_verify },
 	{ "getDefaultDigestName", &pk_getDefaultDigestName },
 	{ "toPEM",         &pk_toPEM },
+	{ "getPrivateKey", &pk_getPrivateKey },
```

Comment:
Isn't this already possible with `:toPEM("private")`?

Reply:
Given the current interface to toPEM, I found it difficult to extend to accept the password argument. I am open to suggestions.